### PR TITLE
Update workflow names in checkinput

### PR DESCRIPTION
**Pull request by guanqiaofeng**
update workflow name `DNA Alignment` to `DNA Seq Alignment`, `RNA Alignment` to `RNA Seq Alignment` to make them consistent with other modules

### CODE REVIEW

File: tests/modules/icgc-argo-workflows/checkinput/nextflow.rnaaln.config [hunk 1]
```diff
@@ -0,0 +1,34 @@
+manifest {
+    name            = 'RNA Seq Alignment'
+    author          = """Linda Xiang"""
```
Comment (edsu7) [inline tests/modules/icgc-argo-workflows/checkinput/nextflow.rnaaln.config:3]:
```suggestion
    author          = """Guangqiao Feng, Linda Xiang"""
```

File: tests/modules/icgc-argo-workflows/checkinput/nextflow.rnaaln.config [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    homePage        = 'https://github.com/icgc-argo-workflows/dnaaln'
+    description     = """ICGC-ARGO analysis workflow for DNA alignment"""
```
Comment (edsu7) [inline tests/modules/icgc-argo-workflows/checkinput/nextflow.rnaaln.config:5]:
```suggestion
    description     = """ICGC-ARGO analysis workflow for RNA alignment"""
```

File: tests/modules/icgc-argo-workflows/checkinput/nextflow.rnaaln.config [hunk 3]
```diff
@@ -0,0 +1,34 @@
+manifest {
+    name            = 'RNA Seq Alignment'
+    author          = """Linda Xiang"""
+    homePage        = 'https://github.com/icgc-argo-workflows/dnaaln'
```
Comment (edsu7) [inline tests/modules/icgc-argo-workflows/checkinput/nextflow.rnaaln.config:4]:
```suggestion
    homePage        = 'https://github.com/icgc-argo-workflows/rnaaln'
```

**Review by edsu7 — CHANGES_REQUESTED** [review]
Minor things, nothing affecting functionality. All tests passed.

File: tests/modules/icgc-argo-workflows/checkinput/nextflow.rnaaln.config [hunk 4]
```diff
@@ -0,0 +1,34 @@
+manifest {
+    name            = 'RNA Seq Alignment'
+    author          = """Guanqiao Feng, Linda Xiang"""
+    homePage        = 'https://github.com/icgc-argo-workflows/rnaaln'
+    description     = """ICGC-ARGO analysis workflow for RNA alignment"""
```
Comment (guanqiaofeng) [inline tests/modules/icgc-argo-workflows/checkinput/nextflow.rnaaln.config:5]: updated fields to address Edmund's comments

**Review by lindaxiang — APPROVED** [review]
LGTM

**Pull request merged**